pkg/statistics/handle/handletest/handle_test.go

```diff
@@ -1512,3 +1512,25 @@ func TestStatsCacheUpdateTimeout(t *testing.T) {
 	require.Equal(t, 6, int(globalStats2.RealtimeCount))
 	require.Equal(t, 2, int(globalStats2.ModifyCount))
 }
+
+func TestLoadStatsForBitColumn(t *testing.T) {
```

Comment:
Could you please add another test case for values other than 0 or 1? Thanks!

Reply:
Done. Add more tests which also includes `"0"` (string) / `"a"` and `2` and `3`...